pkg/interactive/interactive.go

```diff
@@ -177,12 +177,30 @@ func GetOption(input Input) (a string, err error) {
 	if !ok {
 		dflt = ""
 	}
+	defaultMessage := ""
+	if dflt != "" {
+		defaultMessage = fmt.Sprintf("default = '%s'", dflt)
+	}
 	question := input.Question
-	if !input.Required && dflt == "" {
-		question = fmt.Sprintf("%s (optional, choose '%s' to skip selection. "+
-			"The default value will be supplied.)", question, consts.SkipSelectionOption)
+	optionalMessage := ""
+	if !input.Required {
+		optionalMessage = fmt.Sprintf("optional, choose '%s' to skip selection", consts.SkipSelectionOption)
 		input.Options = append([]string{consts.SkipSelectionOption}, input.Options...)
-		dflt = consts.SkipSelectionOption
+		if dflt == "" {
+			dflt = consts.SkipSelectionOption
+		} else {
+			optionalMessage += ". The default value will be provided"
+		}
+	}
+	if optionalMessage != "" || defaultMessage != "" {
+		question += " ("
+		question = fmt.Sprintf("%s%s", question, optionalMessage)
```

Comment:
To the question is added `" ("` and right after that overwritten with the question and optional message. Does not seem right

Reply:
It does not override, it appends to the original variable

Reply:
ahh, the combination of using `Sprintf` and `+=` confused me